BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -454,7 +454,7 @@ $(function () {
   parameters.delegate('.tsk-prm-add', 'click', function () {
     var name = $(this)
       .siblings('.tsk-prm-in')
-      .data('name');
+      .attr('name');
```

Comment:
The problem is that list input elements don't seem to have data associated with them, so .data('name') is unde
![screenshot from 2016-05-16 09-43-40](https://cloud.githubusercontent.com/assets/5174953/15291376/8354afde-1b4b-11e6-951a-8ed9171718ae.png)
fined. See attached screenshot. Also, this code is run only when the user clicks on the "+" button to add a list element, which is only possible when the parameter is enabled when the list is optional. Did I miss anything?